### PR TITLE
Deployment setup

### PR DESCRIPTION
**Pull request by Chulstro**
# SOM-Timer BE Pull Request Form

## Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
`Your message here`

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Other, Please explain:
`Added yml file and hooked up Travis CI`

# Message for reviewer:
`The Gods of CI were kind this day`

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+language: python
+python:
+  - "3.8"
+# command to install dependencies
+install:
+  - pip install -r requirements.txt
```
Comment (sienna-kopf) [inline .travis.yml:6]: SO nice that we built out this .txt file!

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+# command to run tests
+script:
+  - nosetests
```
Comment (sienna-kopf) [inline .travis.yml:9]: Whoops! Nice solve here @Chulstro with realizing that we needed the testing script here.

**Review by sienna-kopf — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (sciencefixion) [reply .travis.yml:6]: Agreed @sienna-kopf, so friendly!

File: .travis.yml (hunk 2 above)
Comment (sciencefixion) [reply .travis.yml:9]: Now I don't have to keep looking this up!

**Review by sciencefixion — APPROVED** [review]

**Review by Abdeboskey — APPROVED** [review]
Y'all are just CRUSHING it.

**Review by jkwest-93 — APPROVED** [review]

**Pull request merged**